### PR TITLE
updated: dates and statuses for milestones

### PR DESCRIPTION
**Pull request by yetson0**
Based on our talk I have updated the dates and statuses of Milestones in `.md` files.
Please check and pull. Cheers.

### CODE REVIEW

**Review by piotr-dziubecki — APPROVED** [review]
lgtm

File: grants/fondant/data.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ Grant-Link: "https://portal.devxdao.com/app/proposal/793",
...
-Current-Milestone: "M3",
-M1: "05.06.2023","Complete",
-M2: "14.09.2023","In review",
-M3: "14.10.2023","In review",
-M4: "21.11.2023","In progress",
-M5: "21.12.2023","To Do",
+Current-Milestone: "M4",
+M1: "05.06.2023","Completed",
+M2: "14.09.2023","Completed",
+M3: "21.10.2023","Completed",
```
Comment (devendran-m) [inline grants/fondant/data.md:13]: Hi @yetson0 ,
Please check with @Anna-defdone  for the revised date.

**Review by devendran-m — CHANGES_REQUESTED** [review]
Hi @yetson0 , 
Please check with @Anna-defdone  for the revised date.

**Comment by yetson0** [comment]
@devendran-m milestone dates updated acc. to @Anna-defdone ; please review.

File: grants/fondant/data.md (hunk 1 above)
Comment (yetson0) [reply grants/fondant/data.md]: changed.

**Review by devendran-m — APPROVED** [review]
Thanks, looking good to me.

**Comment by devendran-m** [comment]
Hi @yetson0 ,  I have updated it in the subsequent PRs. Therefore closing this.

**Pull request closed without merging**